instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -53,9 +54,15 @@
 )
 from opentelemetry.semconv.trace import SpanAttributes
 from opentelemetry.trace import SpanKind, TracerProvider, get_tracer
+from opentelemetry import metrics
+from opentelemetry.sdk.metrics import MeterProvider
```

Comment:
Why is SDK imported here in the instrumentation? Instrumentation never uses SDK.